benchmark/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Re-compiling all the infrastructure all the time is a waste of time, so we
+# pre-compile that part as a library.
+add_library(benchmark Benchmark.cpp Benchmark.h BenchmarkToJson.h BenchmarkToJson.cpp BenchmarkRecordToString.h BenchmarkRecordToString.cpp BenchmarkConfiguration.h BenchmarkConfiguration.cpp BenchmarkMain.cpp)
+target_link_libraries(benchmark absl::flat_hash_map boost_program_options)
```

Comment:
Done, see [fbf5233](https://github.com/ad-freiburg/qlever/pull/860/commits/fbf523324fa5e26903272daa6e0b169481207676).

Funny thing is, I actually only included them, because they **were** in other CMake-files. I thought that, maybe there was some special configuration trick, that needed something like that.